### PR TITLE
Add artwork redirect for logos

### PR DESCRIPTION
**Pull request by hflynn**
Adds redirect for content added to new website on https://github.com/openmicroscopy/www.openmicroscopy.org/pull/205

cc @sbesson

### CODE REVIEW

File: www/tests/check_redirects.py [hunk 1]
```diff
@@ -43,6 +43,7 @@
      '/security/advisories/2014-SV3-csrf/'),
 
     ('/site/support', '/docs'),
+    ('/site/support/ome-artwork/', '/artwork'),
```
Comment (sbesson) [inline www/tests/check_redirects.py]: Trailing `/` in the first initial URL makes the tests fail.

**Comment by hflynn** [comment]
Fix pushed @sbesson

**Comment by sbesson** [comment]
Deployed live

```
(ansible) sbesson@lifesci-76231:prod (master) $ ansible-playbook --ask-become -i ../ansible/inventory/prod-hosts playbooks/www/www.yml 
SUDO password: 
...
PLAY RECAP ********************************************************************************************************************************************
ome-www.openmicroscopy.org : ok=54   changed=6    unreachable=0    failed=0   

(ansible) sbesson@lifesci-76231:prod (master) $ pytest playbooks/www/tests/check_redirects.py
================================================================= test session starts =================================================================
platform darwin -- Python 2.7.14, pytest-3.4.1, py-1.5.2, pluggy-0.6.0
rootdir: /opt/ome/management_tools/prod, inifile:
plugins: testinfra-1.5.5, pyfakefs-3.3
collected 238 items                                                                                                                                   

playbooks/www/tests/check_redirects.py ........................................................................................................ [ 43%]
......................................................................................................................................          [100%]

============================================================= 238 passed in 46.53 seconds =============================================================
(ansible) sbesson@lifesci-76231:prod (master) $ 
```

**Comment by hflynn** [comment]
This doesn't seem to be catching https://www.openmicroscopy.org/site/support/ome-artwork/artwork-usage for some reason :(

**Pull request merged**